packages/manager/apps/pci-kubernetes/src/types/index.ts

```diff
@@ -0,0 +1,33 @@
+export type TKubernetesCluster = {
+  id: string;
```

Comment:
We should synchronize. I named it TKube, and it's located in the api/data/kubernetes.ts file.

Reply:
I have update my type to add your additional properties and rename for your functions